hack/bin/run-local.sh

```diff
@@ -21,9 +29,20 @@ function setEnvFromSecrets {
 
   for specenv in $(kubectl get deployment -n "$NAMESPACE" observability-operator -ojson | jq -c -M '.spec.template.spec.containers[0].env[]') ; do
     envname=$(echo "$specenv" | jq -r '.name')
-    secretname=$(echo "$specenv" | jq -r '.valueFrom.secretKeyRef.name')
-    secretkey=$(echo "$specenv" | jq -r '.valueFrom.secretKeyRef.key')
-    envvalue=$(kubectl get secret -n "$NAMESPACE" "$secretname" -ojson | jq -c -M -r '.data["'"$secretkey"'"]' | base64 -d)
+    # Let's look for a direct value
+    envvalue=$(echo "$specenv" | jq -r '.value')
+
+    # If the value is null, we check if it is a secret reference
+    if [[ "$envvalue" == "null" ]]; then
+      secretname=$(echo "$specenv" | jq -r '.valueFrom.secretKeyRef.name')
+      secretkey=$(echo "$specenv" | jq -r '.valueFrom.secretKeyRef.key')
+      envvalue=$(kubectl get secret -n "$NAMESPACE" "$secretname" -ojson | jq -c -M -r '.data["'"$secretkey"'"]' | base64 -d)
+    fi
+
+    if [[ "$envvalue" == "null" ]]; then
```

Comment:
Well if jq outputs a `null` value, base64 will then decode it into `e`

```
$ echo -n null|base64 -d
e
```

To fix this I would suggest to use a combination of `strings` filter and internal jq `@base64d` function.

```suggestion
      envvalue=$(kubectl get secret -n "$NAMESPACE" "$secretname" -ojson | jq -c -M -r '.data["'"$secretkey"'"] | strings | @base64d')
    fi

    if [[ -z "$envvalue" ]]; then
```